.golangci.yaml

```diff
@@ -0,0 +1,92 @@
+# todo: this should match the configuration of osv-scanner for the most part
```

Comment:
Create an issue for TODOs and link to the issue in the comment.

Reply:
I was thinking it'd be covered by #158 but I'm happy to have a dedicated ticket for it: #274

Reply:
Thanks! Can you change the todos in this doc to the format `# TODO(#274): ` in that case?

Reply:
done! and I've made a note to check if there's a lint for that 😄